src/ProjectNeon/Assets/Features/Effects/PhysicalDamage.cs

```diff
@@ -0,0 +1,15 @@
+﻿public sealed class PhysicalDamage : Effect
+{
+    public int Damage { get; }
+
+    public PhysicalDamage(int damage)
+    {
+        Damage = damage;
+    }
+
+    public void Apply(Member source, Target target)
+    {
+        target.Members[0].State.TakePhysicalDamage(source.State.Attack() * Damage);
```

Comment:
This formula is wrong. 

There should be no multiplier in here. If there is a multiplier, then you need to put the factor, instead of the damage amount.

Reply:
@SilasReinagel Fixed, according to `{ EffectType.PhysicalDamage, e => new SimpleEffect((src, m) => m.TakePhysicalDamage(src.State.Attack() * e.FloatAmount))}`